src/gc/bits.d

```diff
@@ -58,6 +66,7 @@ struct GCBits
         return core.bitop.bt(data, i);
     }
 
+    //pragma(inline,true)
```

Comment:
Was this supposed to be commented out?

Reply:
IIRC inlining didn't always work, but this might be fixed in the meantime. I'll put the pragma back in.